sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -952,6 +952,9 @@ The command status of the event returned will be
 `info::event_command_status::running` once any command group node starts
 executing on a device, and status `info::event_command_status::complete` once
 all the nodes have finished execution.
+
+The queue should be associated with a device and context that are the same
+as the device and context used on creation of the graph.
```

Comment:
Rather than device & context, can this just say the "same context as used on creation of the graph"? This is the error wording that's defined for `handler::ext_oneapi_graph()`, as if the context's are the same then it must contain a valid device.

Reply:
thanks, we may don't need this PR for "queue::ext_oneapi_graph" since the error wording is already there at "handler::ext_oneapi_graph".

Regarding "device & context", the error wording of "begin_recording" is "throws synchronously with error code invalid if recordingQueue is associated with a device or context that is different from the device and context used on creation of the graph.",  it mentions both device and context.  So, shall we change the error wording of "begin_recording" to remove the 'device'? thanks.

Reply:
> thanks, we may don't need this PR for "queue::ext_oneapi_graph" since the error wording is already there at "handler::ext_oneapi_graph".

I don't think there's any implementation/semantic implications of this PR, but if it wasn't obvious to you reading the specification that this error exists, then it won't be obvious to other readers as well. So if duplicating this information in `queue::ext_oneapi_graph()` improves readability, then I think it's still an addition worth making.

>  So, shall we change the error wording of "begin_recording" to remove the 'device'? thanks.

That change makes sense to me for consistency yes.

Reply:
thanks, this PR updated.  And see the change of "begin_recording" part at https://github.com/intel/llvm/pull/11288.

btw, since the context can be created with more than one device, so, does it mean that we only care about the same context, and the different device doesn't matter? thanks.

Reply:
Apologies @guoyejun , I don't think we need #11288.

In my earlier comment I was misremembering the SYCL spec and assuming that a SYCL queue wasn't tied to a particular device, just a context. That isn't the case, so your initial wording here about same context **and** device was correct. If you could also update `handler::ext_oneapi_graph()` in this PR to say "same context and device" in its error wording that would be great.

Reply:
sure, #11288  is closed, and this PR is updated together with the handler part.